components/cosmetic_filters/renderer/cosmetic_filters_js_handler.cc

```diff
@@ -434,41 +408,33 @@ void CosmeticFiltersJSHandler::CSSRulesRoutine(
         blink::BackForwardCacheAware::kAllow);
   }
 
-  if (force_hide_selectors_list &&
-      force_hide_selectors_list->GetList().size() != 0) {
-    std::string json_selectors;
-    if (!base::JSONWriter::Write(*force_hide_selectors_list, &json_selectors) ||
-        json_selectors.empty()) {
-      json_selectors = "[]";
+  base::Value* force_hide_selectors_list =
+      resources_dict->FindListKey("force_hide_selectors");
+  if (force_hide_selectors_list->GetList().size() != 0) {
```

Comment:
perhaps worth to check `force_hide_selectors_list != nullptr` in case we don't have that key.

Reply:
oops, good catch! I thought I had it but I guess not